### PR TITLE
Tdr 528 gitsecrets additional patterns

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: manual/new-starters.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Clone the git-secrets [repository](https://github.com/awslabs/git-secrets) to yo
 Run `sudo make install`.
 This should place git-secrets into your PATH, but if not you may need to move it and run `sudo make install` again OR include the folder made by running `sudo make install` into your PATH.
 
+Then run the following commands to add additional rules to check for AWS account numbers:
+* `git-secrets --add --global '([^0-9])*[0-9]{12}([^0-9])*`: this will add a check for any 12 digit number.
```
Comment (suzannehamilton) [inline manual/new-starters.md:19]: Just to check - have you run this on all the TDR repos to check that false positives aren't a problem in our code?

Comment (TomJKing) [reply manual/new-starters.md:19]: Run against the 31 TDR repos I have locally. Not sure that is all the TDR repos we have now. Didn't get any false positives on these.

File: manual/new-starters.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ Clone the git-secrets [repository](https://github.com/awslabs/git-secrets) to yo
...
+* `git-secrets --add --global --allowed '([^0-9])*123456789012([^0-9])*'`: this will add an allowed dummy AWS account number of *123456789012*. This can be used, if for example, when writing example code in a README which uses an AWS account number.
```
Comment (suzannehamilton) [inline manual/new-starters.md:20]: Sorry, just one more thing: does `git-secrets --add --global --allowed '123456789012'` work as well?

File: manual/new-starters.md (hunk 1 above)
Comment (suzannehamilton) [inline manual/new-starters.md:19]: Just noticed a missing closing `'` quote at the end of the regex.

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**